designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/SupportingText.kt

```diff
@@ -132,29 +143,43 @@ fun SupportingText(
         }
     }
 
+    val density = LocalDensity.current
+    val visibleState = remember {
```

Comment:
Since we are creating the state during composition, this will not get called if the Composable is not included in the tree at all. For example, in the same when we are trying to hide the supporting text. It doesn't seem to animate, it just disappears.